Js/main.js

```diff
@@ -0,0 +1,89 @@
+/* eslint-env jquery */
+
+let myLibrary = [];
+const addNewBookButton = document.querySelector('button.newBook');
+
+function Book(title, author, pages, id) {
+  this.id = id;
+  this.title = title;
+  this.author = author;
+  this.pages = pages;
+}
```

Comment:
## Use es6 arrow syntax
Please convert this typed function to es6 arrow syntax.

Reply:
Changed it